packages/app-elements/src/ui/resources/ResourceListItem/transformers/returns.tsx

```diff
@@ -8,8 +8,14 @@ import { type ResourceToProps } from '../types'
 
 export const returnToProps: ResourceToProps<Return> = ({ resource, user }) => {
   const displayStatus = getReturnDisplayStatus(resource)
+  const returnStockLocationName =
+    resource.order?.market?.name != null
+      ? `to ${resource.order?.market?.name} `
```

Comment:
Question marks are not needed here:

```suggestion
      ? `to ${resource.order.market.name} `
```

Reply:
Makes sense!! :) Fixed!!